tests/core/test_broadcast_config.py

```diff
@@ -51,12 +55,12 @@
 
     tracker = Tracker()
 
-    endpoint1.subscribe(
+    await endpoint1.subscribe(
```

Comment:
This falls into the same category as the `broadcast` that became *async*. We uses this API in a bunch of places that would need to turn into `async` and I actually quite like that synchronous code could use `subscribe` to consume events :)

Reply:
Yeah! I should come back and look at `subscribe`, my idea was that I wanted to use an architecture which got rid of these race conditions once and for all. So `subscribe` blocks until all the inbound connections have confirmed that they're only sending the requested messages. This was supposed to reduce the number of `asyncio.sleep()` I added to the tests. It didn't get rid of very many `asyncio.sleep()` calls though, and I later added the more useful `wait_until_subscribed_to`, so I'll look into simplifying the code and making `subscribe` a regular call again.